### DISCUSSION TITLE
训练时如何加入无标注框的负样本来减少模型误判率

### DISCUSSION THREAD

**Discussion by VJoer** [open-mmlab/mmyolo#702] [Q&A] [answered]
在自定义数据集中训练模型进行测试，总会发现部分无目标的图片出现误判的现象，mmyolo能否支持加入这些负样本到训练集中进行训练？就像[PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection/issues/5956#issuecomment-1124778059)中支持加载无标注框进行训练。

**Comment by hhaAndroid** [comment, marked as answer]
已经支持了。https://mmyolo.readthedocs.io/zh_CN/dev/recommended_topics/training_testing_tricks.html#id6

**Reply by VJoer** [reply]
好的谢谢，那这些纯背景图片是要如何加入到自定义的coco数据集中？就在标注的时候要怎么操作？或者在coco的json中是怎样的格式？
